src/client/nav/request.js

```diff
@@ -134,7 +134,7 @@ spf.nav.request.send = function(url, opt_options) {
         var value = configHeaders[key];
         // Treat undefined and null values as equivalent to an empty string.
         // Note that undefined == null.
-        headers[key] = (value == null) ? '' : value;
+        headers[key] = (value == null) ? '' : String(value);
```

Comment:
Is there a reason not to use `value.toString()`?

Reply:
I could. But this works as well. Any reason to use `toString()` over `String()`?

Reply:
Whatever the JS equivalent is of the phrase "Pythonic". JSonic 🤔?
I just tend to see methods used rather than constructors.

In any case, not sure how active this project is and whether @nicksay wants to accept updates.

Reply:
@nicksay is out until March. I talked to @DavidCPhillips and he seemed OK with the change.

With respect to JSonic - just look to this code base. `String(value)` happens a [couple](https://github.com/youtube/spfjs/blob/988b652f6f979d7445f3160a40e29094030d1be5/src/client/string/string.js#L146) of [times](https://github.com/youtube/spfjs/blob/988b652f6f979d7445f3160a40e29094030d1be5/src/client/net/resource.js#L678). `toString` does not appear to be the convention.

Reply:
fwiw: (null).toString() -> error. String(null) -> "null". Same for undefined and perhaps other obnoxious things.